### PR TITLE
Fix persistence of sort selected when status changed on dashboard page

### PR DESCRIPTION
**Pull request by nathancashmoredwp**
+ some refactoring of tests to use knex seed file.

### CODE REVIEW

File: db/seeds/sort_and_filter/jobs_seed.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        employer: 'Monkey',
+        created_at: '2016-06-29 14:45:27',
+        updated_at: '2016-06-29 14:45:27',
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-14',
+        rating: '1',
+        status: 'applied',
+        status_sort_index: 1,
+      });
```
Comment (lukasz-kaniowski) [inline db/seeds/sort_and_filter/jobs_seed.js]: we could extract jobs array to `fixtures.js` in order to have access to them in tests

File: test/integration/dashboard.spec.js [hunk 2]
```diff
@@ -82,56 +83,78 @@ describe('Dashboard', () => {
         .then(() => dashboardPage.visit(accountId))
         .then(() => expect(dashboardPage.jobCount()).to.equal(0)));
   });
-  describe('should sort job list', () => {
-    const createJob = (attributes) =>
-      new JobsModel(Object.assign({}, jobData, attributes)).save();
+  describe('sort job list', () => {
+    const SEED_ACCOUNT_ID = 'c8330c1f-23f5-4577-943d-151d059af588';
```
Comment (lukasz-kaniowski) [inline test/integration/dashboard.spec.js]: a wee bit magic number, `fixtures.js` would solve this

**Comment by nathancashmoredwp** [comment]
Will look into it ... thought it was fairly explicit seeing as the test references the path to the seed file.

**Comment by lukasz-kaniowski** [comment]
scratch the `fixture.js` idead. I think that extracting `SEED_ACCOUNT_ID` to const and access it in `jobs_seed.js` would be sufficient.

**Pull request merged**